src/components/dialogs/network-modifications/vsc/modification/vsc-modification-dialog.tsx

```diff
@@ -228,10 +228,7 @@ const VscModificationDialog: React.FC<any> = ({
                     .catch((error) => {
                         setVcsToModify(null);
                         setDataFetchStatus(FetchStatus.FAILED);
-                        snackError({
-                            messageTxt: error.message,
-                            headerId: 'VscModificationError',
-                        });
+                        reset(emptyFormData);
```

Comment:
There's no need to reset the form every time we enter the catch block, since no data has been received. The form should only be reset if it already contains data. We can follow the approach used in other modification forms, where resetting is contingent on the presence of pre-existing data

Reply:
i think the reset is done on the others modification, for example `generator-modification-dialog.js:334` or l`oad-modification-dialog.js:159 `